File: src/main/java/alfheim/api/ModInfo.java
package alfheim.api;

public class ModInfo {
	
	public static final String MAJOR = "BETA";
	//public static final String MINOR = "";
	public static final String BUILD = "pre19.1";
	
	public static final String MODID = "alfheim";
	public static final String NAME = "Alfheim";
	public static final String VERSION = MAJOR /*+ "." + MINOR*/ + "-" + BUILD;
	
	public static boolean OBF;
	public static boolean DEV = !OBF;
}

File: src/main/java/alfheim/api/ModInfo.kt
package alfheim.api

object ModInfo {
	
	const val MAJOR = "BETA"
	//const val MINOR = ""
	const val BUILD = "pre19.3"
	
	const val MODID = "alfheim"
	const val NAME = "Alfheim"
	const val VERSION = "$MAJOR-$BUILD"
	
	var OBF: Boolean = false
	var DEV = !OBF
}